How-to-Monitor-Docker-Containers.md

```diff
@@ -55,6 +55,46 @@ ExecStart=/usr/bin/dockerd --containerd=/run/containerd/containerd.sock
 
 My original ExecStart was: `ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock`, note the -H that would cause a duplicate property error.
 
+
+### Method 2 - If you installed docker using snap
```

Comment:
Method 2 is mentioned above. Is this heading intentional?

Do you think an edendum using a `Note` (see https://github.com/orgs/community/discussions/16925) would be better?

Reply:
Didn't know about the "note" it makes perfect sense to use it

Reply:
No, I meant if this might be more readable if the part, that snap stores this file somewere else, should be up where we tell the default path and added as a `Note`

Reply:
Doing so would also remove the need to have a separate section on how the file needs to be modified